src/results/absorption_chiller.jl

```diff
@@ -27,6 +27,24 @@
 # OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 # *********************************************************************************
+"""
+`AbsorptionChiller` results keys:
+- `size_kw` Power capacity size of the absorption chiller system [kW]
+- `size_ton`
+- `year_one_to_tes_series_ton`
+- `year_one_thermal_consumption_series`
+- `year_one_thermal_consumption_kwh`
+- `year_one_thermal_production_tonhour`
+- `year_one_electric_consumption_series`
+- `year_one_electric_consumption_kwh`
+"""
```

Comment:
The docs in this block will not show up in the documentation unless you put it in the triple-quote block above `add_absorption_chiller_results` and put `add_absorption_chiller_results` in the docs/src/reopt/outputs.md .

Reply:
addressed in https://github.com/NREL/REopt.jl/pull/82/commits/a025c77ec34d82542e8e63e45b7588a72a5aa0b9